tools/simde_update.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python
+# Copyright 2020 The Emscripten Authors.  All rights reserved.
+# Emscripten is available under two separate licenses, the MIT license and the
+# University of Illinois/NCSA Open Source License.  Both these licenses can be
+# found in the LICENSE file.
+
+"""Updates the arm_neon.h header taken from SIMDe
+(https://github.com/simd-everywhere/simde) in system/include/neon
+"""
+
+
+import os
+import subprocess
+import sys
+
+from os import path
+
+# Why are the packages being stupid
+# from .shared import get_emscripten_temp_dir
```

Comment:
@sbc100 @kripken Do you know how to properly import `get_emscripten_temp_dir` from `shared` here? Sean and I couldn't figure out how to set up the import properly 😬

Reply:
It looks like other tools in this directory use this magic incantation:
```python
sys.path.insert(1, os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
```
(manually add the toplevel dir to the import path, which we get by getting the dirname of the dirname of the current file)